pom.xml

```diff
@@ -81,6 +81,26 @@
     </dependency>
 
     <dependency>
+      <groupId>edu.amherst.acdc</groupId>
+      <artifactId>acrepo-apple-trees</artifactId>
+      <version>0.1.3</version>
```

Comment:
We may want to externalize the version number as a property... such as: 
```
<properties>
  ...
  <apple-trees.version>0.1.3</apple-trees.version>
  ...
</properties>
```
..but otherwise, it looks good.